zerver/lib/bulk_create.py

```diff
@@ -145,3 +157,124 @@ def bulk_create_streams(realm: Realm,
     Recipient.objects.bulk_create(recipients_to_create)
 
     bulk_set_users_or_streams_recipient_fields(Stream, streams_to_create, recipients_to_create)
+
+def bulk_create_reactions(
+    messages: Iterable[Message] = (),
+    users: Iterable[Any] = (),
+    emojis: Iterable[Any] = ()
+) -> None:
+    messages = list(messages)
+    if not emojis:
+        emojis = _DEFAULT_EMOJIS
+    emojis = list(emojis)
+
+    reactions: List[Reaction] = []
+    for message in messages:
+        reactions.extend(_create_random_reactions_to_message(
+            message, emojis, users))
+    Reaction.objects.bulk_create(reactions)
+
+
+_DEFAULT_EMOJIS = [
+    ('+1', '1f44d'),
+    ('smiley', '1f603'),
+    ('eyes', '1f440'),
+    ('crying_cat_face', '1f63f'),
+    ('arrow_up', '2b06'),
+    ('confetti_ball', '1f38a'),
+    ('hundred_points', '1f4af'),
+]
+
+
+def _create_random_reactions_to_message(
```

Comment:
I renamed this to `add_random_reactions_to_message`, since that feels more natural as an English phrase.